checkbox-support/checkbox_support/scripts/run_watcher.py

```diff
@@ -252,9 +293,11 @@ def report_removal(self, line_str):
         if match:
             logger.info("Mediacard removal test passed.")
 
-        # remove the storage info
-        storage_info_helper(reserve=False, storage_type=self.args.storage_type)
-        sys.exit()
+            # Storage removal info
+            self._storage_info_helper(
```

Comment:
ditto for below, how does this work? it can't right? the function is not defined anywhere and we don't inherit from the usb class